js/flightlog_fields_presenter.js

```diff
@@ -1752,6 +1836,18 @@ function FlightLogFieldPresenter() {
                         default:
                             return (value / 100).toFixed(2);
                     }
+                case 'FAILSAFE':
+                    return value.toFixed(0);
+                case 'GYRO_CALIBRATION':
+                    return value.toFixed(0);
+                case 'ANGLE_MODE':
+                    return value.toFixed(0);
+                case 'ANGLE_TARGET':
+                    return value.toFixed(0);
+                case 'CURRENT_ANGLE':
+                    return value.toFixed(0);
+                case 'DSHOT_TELEMETRY_COUNTS':
+                    return value.toFixed(0);
```

Comment:
We should set the display name of each debug field here, so that it shows up in the log explorer.